### ISSUE TITLE
pycryptodome sub-process never ends.

### ISSUE THREAD

**Issue by bebopimpromptu** [holzschu/Carnets#275]
The installation process <<%pip install yahoofinancials>> fails when collecting pycryptodome sub-process never ends.

**Comment by holzschu** [comment]
That is normal: pycryptodome has **many** modules written in C, and thus cannot be installed inside Carnets, since there are no C compilers on iOS, and only pure Python modules can be installed. a-Shell, another app by the same developer, has pycryptodome already installed.

**Comment by bebopimpromptu** [comment]
Thank you. How can pip install yahoofinancials then? Is there an alternative library? Thanks.

**Comment by holzschu** [comment]
I don't think you can install yahoofinancials, and I do not know enough about it to suggest an alternative library.
